configuration/pih/statusData/mentalHealth/lastDisposition.sql

```diff
@@ -0,0 +1,17 @@
+select concept_from_mapping('PIH', '8620')
+into @dispositionCategory;
+
+
+
+# Get the latest obs_id with dispositionCategory 
+SELECT o.value_coded  into @valueCoded
+FROM obs o
+join encounter e on e.encounter_id = o.encounter_id
+where o.person_id = @patientId
+and o.concept_id = @dispositionCategory
+and o.voided = 0
+order by o.obs_datetime desc
+limit 1;
+
+select
+    @valueCoded as disposition;
```

Comment:
Defer to @dmdesimone , but we likely can use the existing function library that has been developed to make this entire sql statement easier.  All we likely need in this file is something like:

```
select value_coded as disposition from obs where obs_id = latest_obs(@patientId, 'PIH', '8620');
```